From 8c1fc730c7c61d1064a85d39131076e9c2df5a86 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Thu, 24 Oct 2013 14:06:06 -0400
Subject: [PATCH] i7300_edac: fix compile warning
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

To fix:

drivers/edac/i7300_edac.c:825:23: warning: ‘nr_pages’ may be used uninitialized in this function [-Wmaybe-uninitialized]
drivers/edac/i7300_edac.c:781:21: note: ‘nr_pages’ was declared here

we take a one line portion of upstream commit 084a4fccef39ac7abb0
("edac: move dimm properties to struct dimm_info")

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 drivers/edac/i7300_edac.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/edac/i7300_edac.c b/drivers/edac/i7300_edac.c
index f4059e9..487d17a 100644
--- a/drivers/edac/i7300_edac.c
+++ b/drivers/edac/i7300_edac.c
@@ -803,6 +803,7 @@ static int i7300_init_csrows(struct mem_ctl_info *mci)
 	}
 
 	/* Get the set of MTR[0-7] regs by each branch */
+	nr_pages = 0;
 	for (slot = 0; slot < MAX_SLOTS; slot++) {
 		int where = mtr_regs[slot];
 		for (branch = 0; branch < MAX_BRANCHES; branch++) {
-- 
1.8.4.1

